packages/flutter_test/lib/src/browser.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const bool _kIsCompiledToJavaScript = identical(0, 0.0);
+final RegExp _doubleFormatter = RegExp('([0-9]+)\.0');
```

Comment:
You might want to pin this at the end of the word so that it doesn't replace "1.01" with "11". Also you don't need to look for more than one digit preceding.
```suggestion
final RegExp _doubleFormatter = RegExp(r'(\d)\.0\b');
```